props/saber_fett263_buttons.h

```diff
@@ -5924,7 +5987,13 @@ SaberFett263Buttons() : PropBase() {}
             return true;
           }
 #endif
-          if (color_mode_ == CC_COLOR_LIST) {
+          if (
+#ifdef FETT263_REPLACE_CC_COLOR_LIST
```

Comment:
There seems to be a lot of these.
Why not just do something like somewhere higher up:

```
#ifdef FETT263_REPLACE_CC_COLOR_LIST
#define ACTIVE_COLOR_MODE CC_EDIT_COLOR
#else
#define ACTIVE_COLOR_MODE CC_COLOR_LIST
#endif
```

Then here you can just do color_mode_ == ACTIVE_COLOR_MODE

Or something like that?

Reply:
I replaced where it made sense, in other locations the define is changing which functions are used and/or how the mode works as the Color List and Edit Color versions function differently.  They're not interchangeable.